lib/mobility-core/src/Kernel/External/SMS/Interface/Types.hs

```diff
@@ -43,9 +46,20 @@ data SmsServiceConfig = MyValueFirstConfig MyValueFirst.MyValueFirstCfg | Exotel
 data SendSMSReq = SendSMSReq
   { smsBody :: Text,
     phoneNumber :: Text,
-    sender :: Text
+    sender :: Text,
+    smsType :: Maybe SMSType,
```

Comment:
why can't we make it mandatory?
then we don't need this default with empty string `maybe "" toUrlPiece smsType`

Reply:
It is because it is an optional parameter for exotel api so if in case we don't want to pass it then also the exotel api should work